docs/tool/js/util/filter.js

```diff
@@ -152,18 +154,30 @@ var filterUtil = {
 		}
 		filterUtil.filteredData = result;
 	},
-	
-	publishFilteredData: function(){ 
-                                        
+
+	publishFilteredData: function(){
+
         filterUtil.filterData();
 		filterUtil.deduplicate();
 		setState('filteredData', filterUtil.filteredData);
 	},
-    nullsToggleHandler: function(msg){
+  nullsToggleHandler: function(msg){
         var dataChoice = dataChoices.find(function(each){
             return each.source === msg.split('.')[1];
         });
         filterView.filterInputs[dataChoice.short_name].callback(); // fire callback as if the associated filter has been changed
-    }
+  },
+	getActiveFilterValues: function() {
+	    var allFilterValues = filterUtil.getFilterValues();
+			var currentFilters = {};
+			Object.keys(allFilterValues).forEach(function(key){
+				if ( allFilterValues[key][0] ){
```

Comment:
This isn't actually excluding all unused filters. If a filter is set and then cleared (including some that are "set" on load, i.e. all the ones with chloropleth data), `allFilterValues[key][0]` will exist (true) but it will be an empty array `[]`. This actually means it's an inactive filter and should be excluded from this list. Should add another condition to exclude these false positives.